components/docs-chef-io/content/habitat/package_support/naming.md

```diff
@@ -0,0 +1,47 @@
++++
+title = "Habitat package naming conventions"
```

Comment:
This looks like internal documentation and not something that should be customer-facing.